### PR TITLE
adding database to excel capabilities

### PR DESCRIPTION
**Pull request by Obeth297**
Exports an SQL data table to an Excel spreadsheet file.  The arguments required for the method are the SQL table name, and the name of the Excel file you want which is defaulted to EasyPunchCard.

### CODE REVIEW

File: database.py [hunk 1]
```diff
@@ -102,7 +103,7 @@ def update_time(self, additional_minutes:int):
 
     def read_all_users(self):
         if self.admin_status == 0:
-            return "user doesn't have admin status"
+            raise "Error: user doesn't have admin status"
```
Comment (AungA07) [inline database.py:106]: I like changing the return to a raise

File: database.py [hunk 2]
```diff
@@ -126,3 +127,17 @@ def read_user_table(self, student_id):
             cursor.close()
             conn.close()
             return f"{e}"
+        
+    def database_to_excel(self, sql_table_name:str, file_name:str="EasyPunchCard"):
```
Comment (AungA07) [inline database.py:131]: looks good

**Review by AungA07 — APPROVED** [review]
Looks good to me!

**Review by martincarapia — APPROVED** [review]

**Comment by martincarapia** [comment]
Good job, thank you for following my comment.

**Pull request merged**